L/libRadtran/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libRadtran"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.libradtran.org/download/libRadtran-$(version).tar.gz", "eb840e00f2b59648e77775df83d8ae2337880cec155d145228cd65365e3c816f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libRadtran-2.0.4/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+touch lib/libRadtran.so
+make clean
```

Comment:
Why?

Reply:
First, the folder has some extra files which prohibit the normal `make` workflow, so we need to do a `make clean` here. Second, the `make clean` will try to `rm lib/libRadtran.so` which is not existent but it does not use the `-f` option, so we need to touch a pseudo file so that the `make clean` command will not error.